### PR TITLE
WEB-5684 moved to CircleCI

### PR DESCRIPTION
**Pull request by t1gr0u**
#### What are the links to relevant tickets?
[WEB-5684](https://hxshortbreaks.atlassian.net/browse/WEB-5684)
#### What does this PR do?
Moves the build agent from shippable over to CircleCI.
#### What functional/unit tests does this PR have?
N/A
#### How should a developer review this?
Eyes
#### Any background context you want to provide?
You know... shippable and all that
#### Screenshots (if appropriate)
---

#### Quality checklist (for PR author to complete BEFORE code review):
- [x] Bumped version in `package.json`.
- [x] Created `circle.yml` in project.
- [x] Deleted `shippable.yml`.
- [ ] Split any deploy script to `deployToStaging.sh` & `deployToProduction.sh`.
- [x] Setup `GHUSER` environment variable in Circle.
- [x] Setup `GHPASS` environment variable in Circle.
- [x] Setup `GRAPHITE_API_KEY` environment variable in Circle.
- [x] Setup `GRAPHITE_ENDPOINT_PREFIX` environment variable in Circle.
- [x] Setup **slack** webhook.
- [x] Setup API **status** token in Circle.
- [x] Update Readme with new build badge.
- [ ] Added AWS credentials for **shortbreaksBuildAgent**
- [x] Added **shortbreaksBuildAgent** Deploy Key
- [x] I am ready for this to be code reviewed, merged and tested.

#### Reviewer 1 @taijuten
- [x] I agree with the assumptions made in the quality checklist.
- [ ] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

#### Reviewer 2 @lukefrake
- [x] I agree with the assumptions made in the quality checklist.
- [x] I’ve witnessed the work behaving as expected.
- [x] I’ve checked for appropriate test coverage.
- [x] I’ve checked for coding anti-patterns.
- [x] I've checked this work against the requirements of the Jira.
- [x] I don’t believe this work introduces unnecessary technical debt.
- [x] +1 from me!

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,40 +1,41 @@
...
-  },
-  "dependencies": {
-    "transformer": "git+ssh://git@github.com:holidayextras/transformer.git#v2.1.0",
-    "lodash": "2.4.1",
-    "q": "1.0.1",
-    "aws-sdk": "2.0.17"
-  }
+	"name": "the-wall-file-handler-s3",
+	"description": "Handles reading files from amazon s3 for the wall",
+	"version": "1.1.1",
```
Comment (taijuten) [inline package.json]: Should this be a patch, as opposed to a bug fix?

Comment (t1gr0u) [inline package.json]: no code change, just the CI

Comment (notABluesSinger) [inline package.json]: Did patches on all works plugins. I'd say good shout @taijuten.

**Comment by notABluesSinger** [comment]
Also, can we delete the `TODO.md`

**Pull request merged**